README.md

```
@@ -4,13 +4,40 @@
+
+import {ERC7984} from "@openzeppelin/confidential-contracts/token/ERC7984/ERC7984.sol";
+
+abstract contract MyToken is ERC7984 {
+    constructor() ERC7984("MyToken", "MTN", "<CONTRACT-URI>") {
+    }
+}
+```
+
+> [!NOTE]
+> All contracts built using confidentiality must set the coprocessor configuration. This can be done by inheriting config files such as `ZamaEthereumConfig`.
+
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)